### PR TITLE
Refact design tokens and pass it by props

### PR DESCRIPTION
**Pull request by theomoura**
#### What is the purpose of this pull request?

Refact tachyons for design tokens and pass it by props

#### What problem is this solving?

Make style more customizable

#### How should this be manually tested?

[Access this workspace](https://theo--storecomponents.myvtex.com/ninja-300/p)

#### Types of changes

* [x] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [x] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Refact tachyons to use design tokens
```
Comment (lucis) [inline CHANGELOG.md]: Refact to use design tokens*

`tachyons` stayed the same

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+### Changed
```
Comment (lucis) [inline CHANGELOG.md:9]: Add one line between 11 and 10

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- Paddings and margins to match other components.
+- Pass design tokens via props to ProductPrice and ProductName
```
Comment (lucis) [inline CHANGELOG.md]: Passing classes*, not only `design tokens`

**Review by lucis — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Refact to use design tokens
```
Comment (brunojdo) [inline CHANGELOG.md]: This is a feature. 

### Added
- Product Details with Design Tokens ✨

File: react/ProductDetails.js [hunk 5]
```diff
@@ -272,9 +289,6 @@ class ProductDetails extends Component {
                       </div>
                     )}
                   <div className="flex w-100 pv2">
```
Comment (Klynger) [inline react/ProductDetails.js]: I think this `flex` class can be removed

File: react/global.css [hunk 6]
```diff
@@ -7,92 +7,6 @@
...
-
-.vtex-product-details > .vtex-price {
-  height: 125px;
-}
-
-.vtex-product-details .vtex-share {
-  height: 2em;
-}
+.vtex-product-details .vtex-product-name__brand {
+  font-weight: normal !important
```
Comment (Klynger) [inline react/global.css]: It works without `!important`

**Review by Klynger — CHANGES_REQUESTED** [review]

**Review by Klynger — APPROVED** [review]

**Review by lucis — APPROVED** [review]

**Pull request merged**